openapi/SwarmDebug.yaml

```diff
@@ -1,6 +1,6 @@
 openapi: 3.0.3
 info:
-  version: 2.1.0
+  version: 2.2.0
```

Comment:
Revert version change. Also, revert all these changes from the `SwarmDebug` doc. The DebugAPI will be deprecated soon, so no point in adding any new endpoints here.